api/api.go

```diff
@@ -407,24 +407,13 @@ func (b *BlockChainAPI) Call(
 	overrides *StateOverride,
 	blockOverrides *BlockOverrides,
 ) (hexutil.Bytes, error) {
-	if args.To == nil {
-		// todo temporary fix, support empty to address
-		return nil, nil
-	}
-
-	var data []byte
-	if args.Data != nil {
-		data = *args.Data
-	} else if args.Input != nil {
-		data = *args.Input
-	} else {
```

Comment:
This error didn't make much sense, as the transaction args are not required to have any of the two fields populated. The transaction args might as well be about a value transfer, in which case these fields should not be populated. That's why I have removed the error.